### PR TITLE
Corrected some errors using dart sdk m3 0.4.2.8_r20259

### PR DESCRIPTION
**Pull request by giovannicandido**
Some errors and deprecated code preventing toastd for correct function in the SDK version 0.4.2.8_r20259

Added other examples to the example page.

### CODE REVIEW

File: example/toastr.html [hunk 1]
```diff
@@ -13,6 +13,6 @@
     <h1>Example Toast</h1>
 
     <script type="application/dart" src="toastr.dart"></script>
-    <script src="https://dart.googlecode.com/svn/branches/bleeding_edge/dart/client/dart.js"></script>
```
Comment (giovannicandido) [inline example/toastr.html:16]: Deprecated, and slow the application start.

File: pubspec.yaml [hunk 2]
```diff
@@ -5,6 +5,6 @@ authors:
 - Jeff Moss <jmoss@heavyobjects.com>
 homepage: https://github.com/jeffmoss/toastd
 
-dependencies:
-  unittest: { sdk: unittest }
```
Comment (giovannicandido) [inline pubspec.yaml:9]: This package doest not exist

File: pubspec.yaml [hunk 3]
```diff
@@ -5,6 +5,6 @@ authors:
...
+dependencies:	
+  browser: any
```
Comment (giovannicandido) [inline pubspec.yaml:9]: This package contains the dart.js file

**Comment by jeffmoss** [comment]
Thanks for this!

**Pull request merged**